lib/assessment.rb

```diff
@@ -20,20 +25,43 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+    attr_reader :age
+    attr_writer :name, :location
+  def initalize (name, age, location)
+    @name, @age, @location = name, age, location
+  end
+
+  def name
+    @name
+  end
+
+  def age
+    @age
+  end
+
+  def location
+    @location
+  end
```

Comment:
It wasn't necessary to write these - the `attr_...` method generators take care of creating all the methods we need on the fly, when the object gets instantiated.